djangocms_pageadmin/admin.py

```diff
@@ -293,7 +302,7 @@ def changelist_view(self, request, extra_context=None):
         # so we remove it before initiating the standard Django changelist view.
         if 'page_id' in request.GET:
             request.GET = request.GET.copy()
-            del(request.GET['page_id'])
+            del (request.GET['page_id'])
```

Comment:
Was the additional space here intentional?

Reply:
Yeah, the CI failed because there was no space. FIrst time I've encountered it